File: 1035/1035.c
#include <stdio.h>
int main()
{
    int A,B,C,D,E;
    scanf("%d %d %d %d",&A,&B,&C,&D);
    E=A%2;
    if(B > C && D > A && (C+D) > (A+B) && C > 0 && D > 0 && E == 0 )
    {
        printf("Valores aceitos\n");
    }
    else
    {
        printf("Valores nao aceitos\n");
    }
}

File: Python/1035.py
a,b,c,d=[int(x) for x in input().split()]
e=a%2
if b>c and d>a and (c+d)>(a+b) and c>0 and d>0 and e==0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")